src/en/guide/testing/integrationTesting.adoc

```diff
@@ -36,7 +36,8 @@ class ExampleSpec extends Specification {
 
 The `Rollback` annotation ensures that each test method runs in a transaction that is rolled back. Generally this is desirable because you do not want your tests depending on order or application state.
 
-In Grails 3.0 tests rely on `grails.transaction.Rollback` annotation to bind the session in integration tests. Though each test method transaction is rolled back, the `setup()` method uses a separate transaction that is not rolled back.
+In Grails 3.0 tests rely on `grails.transaction.Rollback` annotation to bind the session in integration tests. Though each test method transaction is rolled back, the `setup()` method uses a separate transaction that is not rolled back and the `setup()` method will be executed automatically when the tests is executed.
```

Comment:
I think this `... is not rolled back and the setup()...` should be broken into a sentence like this:

 `is not rolled back. The setup() `